recipes/nf-core/meta.yaml

```
@@ -30,7 +30,9 @@ requirements:
     - requests-cache
     - tabulate
     - git
-    - jsonschema 
+    - jsonschema
+    - rich
+    - inquirer
```

Sadly `inquirer` != `pyinquirer`. Looks like `pyinquirer` might not be on conda yet..?

`setup.py` diff between releases [here](https://github.com/nf-core/tools/compare/1.9...master#diff-2eeaed663bd0d25b7e608891384b7298)

New requirements: 

* jinja2
* PyInquirer==1.0.2
* rich>=4.0.0

```suggestion
    - jinja2
    - rich >=4.0.0
    - pyinquirer=1.0.2
```